src/pages/index.js

```diff
@@ -78,19 +104,129 @@ editProfileButton.addEventListener("click", () => {
   profilePopup.open();
 });
 
-//add new card
+profileImageElement.addEventListener("click", () => {
+  imageChangePopup.open();
+});
+
+/*////////////////////////////////////////////////////////////////////
+/                      Popups                                        /
+////////////////////////////////////////////////////////////////////*/
+
+const viewImagePopup = new PopupWithImage(".view-image-popup");
+const confirmDeletePopup = new PopupWithForm({
+  popupSelector: ".delete-card-popup",
+  handleFormSubmit: () => {
+    deleteCard(cardElement);
+  },
+});
 
 const addCardPopup = new PopupWithForm({
   popupSelector: ".add-card-popup",
   handleFormSubmit: (formData) => {
-    renderCard(formData);
-    addCardPopup.close();
+    addCardPopup.setSavingMessage();
+    api
+      .addCard(formData)
+      .then((data) => {
+        renderCard(data);
+        addCardPopup.close();
+        addCardPopup.resetSubmitMessage("Create");
```

Comment:
`        addCardPopup.resetSubmitMessage("Create");
`
You need to return the **default** button text only in blocks `finally` so that the button could get the text back in any case.


**Please, correct it everywhere**